extensions/amp-lightbox/0.1/amp-lightbox.js

```diff
@@ -164,6 +174,23 @@ class AmpLightbox extends AMP.BaseElement {
   }
 
   /**
+   * reschedule layout on animation/transition end. But ignore the first transitionend event
+   * @param {!Event} event
+   */
+  reschedule_(event) {
+    if (event.type == 'transitionend') {
+      // Ignore the first transitionend event.
```

Comment:
(a) can we be more specific, i.e. by looking into `event.target` or specific properties? 
(b) given debounce don't we run a risk of missing the followup events?
(c) is this a bit deal if we proceed an extra time? I think runtime does additional debouncing naturally, right?